conf/shiro.ini

```diff
@@ -60,10 +60,14 @@ role2 = *
 role3 = *
 
```

Comment:
Can you add one more role admin i.e. `admin = *` here, so that other user can also use that role.